src/test/java/org/broadinstitute/consent/http/AbstractTestHelper.java

```diff
@@ -5,6 +5,10 @@
 
 public abstract class AbstractTestHelper {
 
+  public static boolean disableTestContainers() {
+    return System.getProperty("disableTestContainers", "false").equals("true");
+  }
```

Comment:
```suggestion
  public static boolean disableTestContainers() {
    String defaultProp = "false";
    return System.getProperty("disableTestContainers", defaultProp).equals("true");
  }
```

Clarifies that the second argument is a default for `getProperty`, otherwise I found it hard to read.